src/visualization.rs

```diff
@@ -0,0 +1,68 @@
+use crate::issues;
+use geojson::Value::Point;
+use geojson::{Feature, FeatureCollection};
+use gtfs_structures::{Gtfs, ObjectType};
+use std::sync::Arc;
+
+pub fn add_issue_visualization(issue: &mut issues::Issue, gtfs: &Gtfs) {
+    match issue.object_type {
+        Some(ObjectType::Stop) => {
+            let stop_id = issue.object_id.clone();
+            let stop_feature = geojson_feature_point(&stop_id, gtfs);
```

Comment:
instead of `[&stop_id].iter()` you can use [iter::once](https://doc.rust-lang.org/std/iter/fn.once.html) to do something like `std::iter::once(&stop_id)`